src/modules/audio/audio.c

```diff
@@ -524,7 +472,15 @@ void lovrAudioSetCaptureFormat(SampleFormat format, int sampleRate)
 
 void lovrAudioUseDevice(AudioType type, const char *deviceName) {
   free(state.config[type].deviceName);
-  state.config[type].deviceName = strdup(deviceName);
+
+#ifdef ANDROID
+  // XX<nevyn> miniaudio doesn't seem to be happy to set a specific device an android (fails with
+  // error -2 on device init). Since there is only one playback and one capture device in OpenSL,
+  // we can just set this to NULL and make this call a no-op.
```

Comment:
Strange.  If getDevices returns valid devices but they don't work with useDevice, maybe we should file this with miniaudio.

Reply:
yes you're right. There seems to be two different bugs on Quest and Quest 2 too. Quest 1 returns _no_ capture devices even though it clearly has one; and Quest 2 has the above bug. I should file them with miniaudio. Until it's fixed upstream we'll need this workaround though. I can make sure to remove the workaround once upstream has a fix